charts/kong-collectorapi/ci/default-values.yaml

```diff
@@ -0,0 +1,2 @@
+image:
+  repository: registry.kongcloud.io/kong-brain-immunity
```

Comment:
This CI should use Bintray, the source our customers use for brain and immunity.

Reply:
oh this didn't work before I'll give it another try

Reply:
it seems as if the bintray and kind incompatibility is still an issue. This is why I'm using kongcloud registry.
![image](https://user-images.githubusercontent.com/3679927/80637567-810f2100-8a5f-11ea-81a8-717a66f73cca.png)
https://www.jfrog.com/jira/browse/RTFACT-20170
it says resolved in January but I remember this not working last time i tried it February too.

Reply:
I can try to workaround the workaround with kind load I guess